google-storage.yml

```diff
@@ -15,7 +15,7 @@
 version: 1
 name: csb-google-storage-bucket
 id: b247fcde-8a63-11ea-b945-cb26f061f70f
-description: Google Cloud Storage that uses the Terraform back-end and grants service accounts IAM permissions directly on the bucket.
+description: Google Cloud Storage that uses OpenTofu and grants service accounts IAM permissions directly on the bucket.
```

Comment:
I'm not sure that this description was accurate before the change. Does the cloud storage use a Terraform back-end or OpenTofu? I don't think it does.

Reply:
After reading again the sentence and thinking about your comment, I decided to reword the paragraph. Thank you very much, @blgm.